charts/datadog/values.yaml

```
@@ -510,6 +510,15 @@ datadog:
     iast:
       # datadog.asm.iast.enabled -- Enable Application Security Management Interactive Application Security Testing by injecting `DD_IAST_ENABLED=true` environment variable to all pods in the cluster
       enabled: false
+  ## OTel collector related configuration
+  otelcollector:
```

we usually use camel case.
Also do we want to have a more generic name like `otel` 

```suggestion
  otelCollector:
```

---

`otel` is too generic. We want to use collector to indicate that it is collector that is running with the agent

---

but for futur evolution, do we want to make it visible? or do we plan to even improve the OTEL integration in the agent?
No strong option but the shortest the field names are, the less user can do typo. 
My suggestion about camel case is still valid.

---

I have updated to use the Camel case. I don't see reason if we should run anything other than the collector in the Agent. Any OTEL integration we enable will be part of OTel Collector.